app/presenters/individual_statistics_presenter.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+require "#{Rails.root}/lib/word_count"
+
+#= Presenter for users view
```

Comment:
A more precise description would be helpful here. The key point is that this class is intended to provide statistics about a user's contributions *during courses in which the user was a student*.